.github/workflows/remote-bazel-build.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Test
         run: |
-          bb remote test //... \
+          bb remote --env=GIT_REPO_DEFAULT_BRANCH=master test //... \
```

Comment:
revert before merging?

Reply:
The remote bazel code determines the default branch from the local git state. The github runner only checks out one commit, so we'd have to do that hacky thing where we briefly checkout master in order to take advantage of that. It's easier to just explicitly pass the flag